pages/use_cases/life-monitor.md

```diff
@@ -1,4 +1,5 @@
 ---
+page_id: life-monitor
```

Comment:
good idea to standardise on `-` separators, would just add a redirect in case anyone has the old path bookmarked
```suggestion
page_id: life-monitor
redirect_from: life_monitor
```